migrate_pim/migrate_to_saas.rst

```diff
@@ -0,0 +1,157 @@
+Migrate to Serenity (SaaS)
+==========================
+
+`Learn more about Akeneo's SaaS offering (Serenity) <https://help.akeneo.com/en_US/everything-you-need-to-know-about-our-pim-versions#serenity>`_
+
+If you are using **Enterprise Edition** of an On-Premise or Flexibility (PaaS) instance, your instance can be migrated to Serenity!
+
+What to expect
+--------------
+
+* Your PIM's URL will not change if you are migrating a Flexibility (PaaS) instance
+
+* If you have multiple instances (*ex: testing, staging, and production*), each instance needs to be migrated individually
```

Comment:
It's also possible to use the production data to create all the environments if they want.